config/airkeeper.example.json

```diff
@@ -5,30 +5,56 @@
       }
     }
   ],
   "triggers": {
     "rrpBeaconServerKeeperJobs": [
       {
         "chainIds": ["31337"],
-        "templateId": "0x50c604914d8ed35473149457a1a0912b785813b4e2e51bd2b75409ca25c50e1d",
-        "templateParameters": [
-          { "type": "bytes32", "name": "to", "value": "USD" },
-          { "type": "bytes32", "name": "_type", "value": "int256" },
-          { "type": "bytes32", "name": "_path", "value": "result" },
-          { "type": "bytes32", "name": "_times", "value": "1000000" }
-        ],
-        "overrideParameters": [
-          { "type": "bytes32", "name": "from", "value": "ETH" }
-        ],
+        "templateId": "0x6f737bbf31dfed584a16f53b7d725ff64bee67e79a468259456fb40aa19c60c4",
+        "overrideParameters": [{ "type": "string32", "name": "from", "value": "ETH" }],
         "oisTitle": "Currency Converter API",
         "endpointName": "convertToUSD",
         "deviationPercentage": "5",
         "keeperSponsor": "0x2479808b1216E998309A727df8A0A98A1130A162",
         "requestSponsor": "0x61648B2Ec3e6b3492E90184Ef281C2ba28a675ec"
       }
+    ],
+    "proto-psp": [
+      {
+        "subscriptionId": "0xdf6d2785e1fc71ba3aa08fccb77d74ac4ba678208bc33c445152ea999a0cc8af",
+        "overrideParameters": [{ "type": "string32", "name": "from", "value": "ETH" }],
+        "oisTitle": "Currency Converter API",
+        "endpointName": "convertToUSD"
+      }
     ]
+  },
+  "subscriptions": {
+    "0xdf6d2785e1fc71ba3aa08fccb77d74ac4ba678208bc33c445152ea999a0cc8af": {
+      "chainId": "31337",
+      "airnodeAddress": "0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace",
+      "templateId": "0x6f737bbf31dfed584a16f53b7d725ff64bee67e79a468259456fb40aa19c60c4",
+      "parameters": "0x",
+      "conditions": "0x31624200000000000000000000000000000000000000000000000000000000005f636f6e646974696f6e46756e6374696f6e4964000000000000000000000000dc96acc8000000000000000000000000000000000000000000000000000000005f636f6e646974696f6e506172616d657465727300000000000000000000000000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000989680",
+      "relayer": "0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace",
+      "sponsor": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
+      "requester": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0",
+      "fulfillFunctionId": "0x206b48f4"
+    }
+  },
+  "templates": {
+    "0x6f737bbf31dfed584a16f53b7d725ff64bee67e79a468259456fb40aa19c60c4": {
+      "endpointId": "0x13dea3311fe0d6b84f4daeab831befbc49e19e6494c41e9e065a09c3c68f43b6",
+      "templateParameters": [
+        { "type": "string32", "name": "to", "value": "USD" },
+        { "type": "string32", "name": "_type", "value": "int256" },
+        { "type": "string32", "name": "_path", "value": "result" },
+        { "type": "string32", "name": "_times", "value": "1000000" }
+      ]
```

Comment:
I can do that but the thing is that I tried to put the exact same fields that would be saved on-chain and in the case we remove them from here switching to those values instead of reading them from config would be easier. Are you thinking that we could use the encoded values directly to speed things up? is this field in subscription be ever used?

Reply:
You will have to encode these to verify the template ID anyway so keeping them decoded doesn't actually save time. I'm just saying that either both should be decoded or both should be encoded. I prefer encoded because it's less ambiguous, people may try to change the order of decoded parameters for example and that would change the template/subscription IDs.